node/src/api-client.js

```diff
@@ -1,23 +1,26 @@
 const axios = require('axios');
-
-const apiBaseUrl = process.env.ENV0_API_URL || 'https://api.env0.com';
+const { version } = require('../package.json');
 
 class Env0ApiClient {
   async init(apiKey, apiSecret) {
     const auth = `${apiKey}:${apiSecret}`;
     const buff = Buffer.from(auth).toString('base64');
-    this.authHeader = { Authorization: `Basic ${buff}` };
+
+    this.apliClient = axios.create({
+      baseURL: process.env.ENV0_API_URL || 'https://api.env0.com',
+      headers: {
+        Authorization: `Basic ${buff}`,
```

Comment:
this could be nicer using 
`
auth: {
    username: apiKey,
    password: apiSecret
  }
`
instead of creating the header ourselves 

not important AT ALL of course

Reply:
1ca2543